generator.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+echo "What do you want your project to be called?"
+
+read project_name
+
+mkdir $project_name
```

Comment:
I'm ok with that.  Could we then print the project name we'll be generating with and ask the user if that's ok?  If not, just bail.  Although I guess it's easy enough to just `rm -rf` the directory and re-run the generator.sh